### ISSUE TITLE
Please provide standalone exe

### ISSUE THREAD

**Issue by igorgatis** [dotnet/codeformatter#162]
Downloaded the binary release and when I tried running it complained about missing dlls which seem to belong to vs 2015.

Is it correct to say that CodeFormatter tool requires vs 2015 installation? If that is true... That's far too inconvenient.

For companies like mine, which would.love to use the tool but can't migrate right the way to vs 2015, could you please provide a standalone exe (e.g. no external deps)?

**Comment by hickford** [comment]
https://github.com/dotnet/codeformatter/releases/tag/v1.0.0-alpha5 worked for me, you need to extract the whole folder though

**Comment by khellang** [comment]
I don't think you need to install full VS2015 though. [Microsoft Build Tools 2015](http://www.microsoft.com/en-us/download/details.aspx?id=48159) should be sufficient, but I agree; there should be a standalone .exe available.

**Comment by paulvanbrenk** [comment]
A nuget package would be great too.

**Comment by terrajobst** [comment]
I believe the problem is the MSBuild dependency in order to parse and evaluate the project.

Out of curiosity: what's the reason you don't have MSBuild? Is it because you have an older version or is it because you run non-Windows?

**Comment by ghuntley** [comment]
Agreed, would be fantastic to have CodeFormatter distributed as a NuGet package.
